module/Frontpage/src/Frontpage/Form/Page.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace Frontpage\Form;
+
+use Zend\Form\Form;
+use Zend\InputFilter\InputFilter;
+use Zend\I18n\Translator\TranslatorInterface as Translator;
+
+class Page extends Form
+{
+
+    public function __construct(Translator $translator)
+    {
+        parent::__construct();
+
+        $this->add(array(
+            'name' => 'category',
+            'type' => 'text',
+        ));
+
+        $this->add(array(
+            'name' => 'subCategory',
+            'type' => 'text',
+        ));
+
+        $this->add(array(
+            'name' => 'name',
+            'type' => 'text',
+        ));
+
+        $this->add(array(
+            'name' => 'dutchTitle',
+            'type' => 'text',
+            'options' => array(
+                'label' => $translator->translate('Dutch title')
+            )
+        ));
+
+        $this->add(array(
+            'name' => 'englishTitle',
+            'type' => 'text',
+            'options' => array(
+                'label' => $translator->translate('English title')
+            )
+        ));
+
+        $this->add(array(
+            'name' => 'dutchContent',
+            'type' => 'text',
+            'options' => array(
+                'label' => $translator->translate('Dutch content')
+            )
+        ));
+
+        $this->add(array(
+            'name' => 'englishContent',
+            'type' => 'text',
+            'options' => array(
+                'label' => $translator->translate('English content')
+            )
+        ));
+
+        $this->add(array(
+            'name' => 'requiredRole',
+            'type' => 'text',
+            'options' => array(
+                'label' => $translator->translate('Required role'),
+                'value' => 'guest'
+            )
+        ));
+
+        $this->add(array(
+            'name' => 'submit',
+            'type' => 'submit',
+            'attributes' => array(
+                'value' => $translator->translate('Save')
+            )
+        ));
+
+        $this->initFilters();
+    }
+
+    protected function initFilters()
```

Comment:
It is better to use the `InputFilterProviderInterface` here. See https://github.com/GEWIS/gewisweb/blob/master/module/Decision/src/Decision/Form/Notes.php for an example.